src/test/java/core/basesyntax/RegistrationServiceTests.java

```diff
@@ -0,0 +1,147 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.InvalidDataException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class RegistrationServiceTests {
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+
+    @BeforeAll
+    public static void set_up() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @Test
+    public void register_validUser_success() {
+        User newUser = new User();
+        newUser.setLogin("Bob679");
+        newUser.setAge(18);
+        newUser.setPassword("0987654321");
+        registrationService.register(newUser);
+        User expectedUser = storageDao.get(newUser.getLogin());
+        assertEquals(expectedUser.getLogin(), newUser.getLogin());
+        assertEquals(expectedUser.getPassword(), newUser.getPassword());
+        assertEquals(expectedUser.getAge(), newUser.getAge());
+        assertEquals(storageDao.size(), 1);
```

Comment:
Your test `register_validUser_success` should also include edge cases for the user's age and password length to ensure the edge cases are handled correctly.